styles/index.css

```diff
@@ -0,0 +1,61 @@
+body {
+  background-color: powderblue;
+}
+body {
+  margin: 40px;
+}
+
+.wrapper {
+  display: grid;
+  grid-gap: 10px;
+  color: #fff;
+}
+
+.box {
+  background-color: #fff;
+  color: #000;
+  border-radius: 5px;
+  padding: 20px;
+  font-size: 150%;
+}
+
+.city {
+  grid-column: 1 / 3;
+  grid-row: 1;
+}
+.garden {
+  grid-column: 3;
+  grid-row: 1 / 3;
+}
+.temp {
+  grid-column: 1;
+  grid-row: 2;
+}
+.sky {
+  grid-column: 2;
+  grid-row: 2;
+}
+
+.clickable {
+  cursor:pointer;
+}
+
+.red {
```

Comment:
Styles like this are fine, but prefer css class names that provide information about the role it is fulfilling, rather than a description of how it looks.